src/components/EnterpriseApp/data/enterpriseCurationReducer.js

```diff
@@ -54,12 +71,77 @@ function getHighlightSetsFromState(state) {
   return state.enterpriseCuration?.highlightSets || [];
 }
 
+/**
+ * Helper function to determine if a content is archived.
+ *
+ * @param {Object} content
+ * @returns {Boolean}
+ */
+export function isArchivedContent(content) {
+  const hasArchivedCourseRun = content.courseRunStatuses?.includes(COURSE_RUN_STATUSES.archived);
```

Comment:
nit: might be an opportunity for an early return if `courseRunStatuses` isnt always set?

```jsx
const { courseRunStatuses } = content;
if (!courseRunStatuses) {
  return false;
}
```

Then, you shouldn't need to include the `?.` when accessing the `courseRunStatuses` variable below.

---

I'm also wondering if the below logic could be a bit more concise but still have the same semantic meaning? Perhaps along the lines of the following, where the check is to see if _all_ elements are either `archived` or `published`:

```jsx
const ARCHIVABLE_STATUSES = [COURSE_RUN_STATUSES.archived, COURSE_RUN_STATUSES.unpublished];
return courseRunStatuses.every(status => ARCHIVABLE_STATUSES.includes(status));
```

Examples:

```jsx
["archived"] // true
["unpublished"] // true
["archived", "unpublished"] // true
["archived", "published", "unpublished] // false
```

Reply:
thanks for this suggestion!